src/main/java/com/spotinst/sdkjava/model/bl/oceanCD/RolloutSpecStrategy.java

```diff
@@ -0,0 +1,84 @@
+package com.spotinst.sdkjava.model.bl.oceanCD;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RolloutSpecStrategy {
+    @JsonIgnore
+    private Set<String>                               isSet;
+    private String                                    name;
+    private List<BackgroundVerificationArgs>          args;
```

Comment:
Should it be RolloutSpecArgs?

Reply:
I am re-using the same class used in Strategy